src/server/services/kvk.ts

```diff
@@ -114,13 +114,26 @@ export function getKvkAddress(kvkData: KVKData) {
   }
 
   if (vestigingen.length) {
-    const vestiging = kvkData?.vestigingen.find(
+    const hoofdVestiging = kvkData?.vestigingen.find(
       (vestiging) =>
-        !!vestiging.bezoekadres &&
-        (vestiging.bezoekadres.mokum === true ||
-          vestiging.bezoekadres.woonplaatsNaam === 'Amsterdam')
+        vestiging.isHoofdvestiging &&
```

Comment:
Even alleen naar de code kijkend, we hebben tegewoordig de `?.` operator! En misschien kun je, voor de duidelijkheid van het bepalen van het adres de dingen beter uitschrijven:

```
const vestigingenMokum = vestingen.filter(vestiging => vestiging.bezoekadres?.mokum || vestiging.postadres?.mokum);
const hoofdVestiging = vestigingenMokum.find(vestiging => isHoofdvestiging);
const eersteDeBesteVestiging = vestigingenMokum?.[0];

const adres = 
     || hoofdVestiging?.bezoekadres
     || hoofdvestiging?.postadres
     || eersteDeBesteVestiging?.bezoekadres
     || eersteDeBesteVestiging?.postadres;
```